network_wrangler/roadway/network.py

```diff
@@ -330,10 +330,13 @@ def apply(self, project_card: Union[ProjectCard, dict]) -> RoadwayNetwork:
         if not (isinstance(project_card, ProjectCard) or isinstance(project_card, SubProject)):
             project_card = ProjectCard(project_card)
 
-        project_card.validate()
+        # project_card.validate()
+        if not project_card.valid:
+            WranglerLogger.error("Invalid Project Card: {project_card}")
+            raise ValueError(f"Project card {project_card.project} not valid.")
```

Comment:
project_card.validate() didn't work, jsondata doesn't have `project` in sub projects

Reply:
@e-lo the `selection-refactor` branch is still crashing here.